### PR TITLE
[ADP-3417] Add a deposit wallet UI empty http service

### PR DESCRIPTION
**Pull request by paolino**
This PR prepare the ground for the deposit wallet UI by 

- moving the personal UI code to its own `Personal` namespace
- create a `Deposit` UI namespace and populate with a splash page
- add flags to control the presence of the deposit UI port and service
- remove unused code
- document the common code

Both http UI will be available on 8091 and 8092 running i.e.
```
cd run/preprod/nix
DEPOSIT_WALLET_UI_PORT=8092 WALLET_UI_PORT=8091 ./run.sh start
```

ADP-3417

### CODE REVIEW

File: docs/site/src/user/cli.md [hunk 1]
```diff
@@ -79,6 +80,12 @@ Available options:
   --random-port            serve wallet API on any available port (conflicts
                            with --port)
   --port INT               port used for serving the wallet API. (default: 8090)
+  --ui-random-port         serve the personal wallet UI on any available port
+                           (conflicts with --ui-port)
+  --ui-port INT            port used for serving the personal wallet UI.
+  --ui-deposit-random-port serve the deposit wallet UI on any available port
```
Comment (paweljakubas) [inline docs/site/src/user/cli.md:86]: :+1:

File: lib/exe/app/cardano-wallet.hs [hunk 2]
```diff
@@ -229,7 +230,8 @@ data ServeArgs = ServeArgs
     { _hostPreference :: HostPreference
     , _mode :: Mode CardanoNodeConn
     , _listenApi :: Listen
-    , _listenUi :: Maybe Listen
+    , _listenPersonalUi :: Maybe Listen
```
Comment (paweljakubas) [inline lib/exe/app/cardano-wallet.hs]: let's call it `shelley` wallet

File: lib/ui/src/Cardano/Wallet/UI/Deposit/API.hs [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    , (:<|>) (..)
+    , (:>)
+    )
+
+import qualified Data.ByteString.Lazy as BL
+
+-- | Pages endpoints
+type Pages =
+    "about" :> SessionedHtml Get
+        :<|> "network" :> SessionedHtml Get
```
Comment (paweljakubas) [inline lib/ui/src/Cardano/Wallet/UI/Deposit/API.hs:43]: should we be compatible with what we have https://github.com/cardano-foundation/cardano-wallet/blob/master/lib/customer-deposit-wallet/http/Cardano/Wallet/Deposit/HTTP/Types/API.hs ? (of course besides `about` and `settings`) ?

**Review by paweljakubas — APPROVED** [review]
Nice refactoring. Very fond of of reuse `Cardano.Wallet.UI.Common.*` modules between two UIs. I would prefer `shelley` instead of `personal` naming though.

File: lib/ui/src/Cardano/Wallet/UI/Deposit/API.hs (hunk 3 above)
Comment (paolino) [reply lib/ui/src/Cardano/Wallet/UI/Deposit/API.hs:43]: Not at the page level, but at the data level, see the `Data` type.

**Pull request merged**